src/jllvm/class/Descriptors.hpp

```diff
@@ -129,6 +129,8 @@ class FieldType
             default: return false;
         }
     }
+
+    constexpr static FieldType fromPretty(std::string_view text);
```

Comment:
Could you leave a short doc comment describing how this differs from the normal `FieldType` constructor?